imageeditlibrary/src/main/res/values/strings.xml

```diff
@@ -1,54 +1,54 @@
-
-    <string name="no_choose">please choose a image for edit</string>
-    <string name="auto_newline">AutoNewline</string>
+    <string name="app_name">Edit image</string>
+    <string name="no_images">No found the image</string>
+    <string name="images">%d Images</string>
+    <string name="save">Save</string>
+    <string name="apply">Apply</string>
+
+    <string name="stickers">Stickers</string>
+    <string name="filter">Filter</string>
+    <string name="crop">Crop</string>
+    <string name="rotate">Rotate</string>
+    <string name="text">Text</string>
+    <string name="edit">Draw</string>
+
+
+
+    <string name="materialcolorpicker__app_name" translatable="false">Android Material Color Picker Dialog</string>
+    <string name="materialcolorpicker__descLib" translatable="false">Material Color Picker Dialog</string>
+    <string name="materialcolorpicker__errHex" translatable="false">Hex Code Format incorrect</string>
+    <string name="materialcolorpicker__inputColor" translatable="false">Color:</string>
+    <string name="materialcolorpicker__btnSelectColor" translatable="false">Select</string>
+    <string name="saving_image">Save...</string>
+    <string name="input_hint">Input text ...</string>
+
+    <string name="no_choose">Please choose a image for edit</string>
+    <string name="auto_newline" translatable="false">AutoNewline</string>
     <string name="save_error">Save error!</string>
-    <string name="handing">loading...</string>
+    <string name="handing">Loading...</string>
 
     <string-array name="filters">
-        <item>origin</item>
-        <item>soft</item>
-        <item>black</item>
-        <item>classic</item>
-        <item>gorgeous</item>
-        <item>retro</item>
-        <item>grace</item>
-        <item>film</item>
-        <item>memory</item>
-        <item>yogurt</item>
-        <item>fleeting</item>
-        <item>shine</item>
-        <item>mosaic</item>
+        <item>Origin</item>
+        <item>Soft</item>
+        <item>Black</item>
+        <item>Classic</item>
+        <item>Gorgeous</item>
+        <item>Retro</item>
+        <item>Grace</item>
+        <item>Film</item>
+        <item>Memory</item>
+        <item>Yogurt</item>
+        <item>Fleeting</item>
+        <item>Shine</item>
+        <item>Mosaic</item>
     </string-array>
 
-    <string name="more">more</string>
+    <string name="more">More</string>
     <string name="exit_without_save">The edited image is not saved. Do you want to exit? </string>
-    <string name="confirm">confirm</string>
-    <string name="cancel">cancel</string>
+    <string name="confirm">Confirm</string>
+    <string name="cancel">Cancel</string>
 
-    <string name="beauty">beauty</string>
-    <string name="smooth">smooth</string>
-    <string name="white_skin">white skin</string>
+    <string name="beauty">Ceauty</string>
+    <string name="smooth">Smooth</string>
+    <string name="white_skin">White skin</string>
```

Comment:
```suggestion
    <string name="smooth">Smooth</string>
```

Reply:
There are some files in the build folder that are generated and should not be edited and use these references.
Removed from translations of: de, es, fr, no, pt.